### DISCUSSION TITLE
Error while migrate the project

### DISCUSSION THREAD

**Discussion by kumarishalini6** [Praful932/Kitabe#181] [Q&A]
Hii, I was trying to migrate the project but it shows some error
![16176929209757572007680437321368](https://user-images.githubusercontent.com/69043944/113672210-0b7eec80-96d5-11eb-9fb6-2ff409b9624b.jpg)

**Comment by Praful932** [comment]
Your fork is behind @kumarishalini6 check out the [starter guide](https://github.com/Praful932/Kitabe/discussions/42) FAQs :)
